### PR TITLE
CPLAT-4192: Update json_schema to be Dart 2 compatible

### PR DESCRIPTION
**Pull request by sebastianmalysa-wf**
## Overview:

Update ```json_schema``` to be Dart 2 compatible.  

## Testing suggestions:

- [ ] CI passes on Dart 1 and Dart 2  

## Potential areas of regression:

- None, but it might be worth running a few consumer tests. 

---

> __FYA:__ @michaelcarter-wf @corwinsheahan-wf @evanweible-wf @seanburke-wf @smaifullerton-wk

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: .travis.yml [hunk 1]
```diff
@@ -2,44 +2,17 @@ language: dart
 dist: precise
```
Comment (evanweible-wf) [inline .travis.yml]: @sebastianmalysa-wf try removing this line, I think that will do it

**Comment by semveraudit-wf** [comment]
### Public API Changes

 No changes to the public API found for commit db63a87ffbbbe736ce3f628c4559a61e065852fa

Showing results for db63a87ffbbbe736ce3f628c4559a61e065852fa
> Powered by [semver-audit-service](https://github.com/Workiva/semver-audit-service). Please report any problems by [filing an issue](https://github.com/Workiva/semver-audit-service/issues).
> Reported by the dart semver audit client 2.0.0
[Browse public API](https://semver-audit.wk-dev.wdesk.org/Workiva/json_schema/db63a87ffbbbe736ce3f628c4559a61e065852fa).

**Comment by michaelcarter-wf** [comment]
+10 

- [x] CI Passes on D1 and D2

**Comment by michaelcarter-wf** [comment]
@Workiva/release-management-p

**Pull request merged**